include/rosflight.h

```diff
@@ -58,6 +60,7 @@ class ROSflight
   ROSflight(Board& board, CommLinkInterface& comm_link);
 
   Board &board_;
+  MemoryManager memory_manager_;
   CommManager comm_manager_;
 
   Params params_;
```

Comment:
Nit picking here, but could we group all 3 configuration-related things right here?:

```c++
MemoryManager memory_manager_;
Params params_;
ConfigManager config_manager_;
```